.github/workflows/ohos.yml

```diff
@@ -52,12 +52,34 @@ jobs:
         run: sudo apt update && python3 ./mach bootstrap --skip-lints
       - name: Setup OpenHarmony SDK
         id: setup_sdk
-        uses: openharmony-rs/setup-ohos-sdk@v0.1
+        uses: openharmony-rs/setup-ohos-sdk@v0.1 # TODO: this needs to be updated
```

Comment:
@jschwe could you publish a new version of the action for this.

Reply:
Published v0.1.2 and redirected v0.1 to point to v0.1.2

Reply:
Thanks! I've removed the todo.